lib/utils/getTreasuryBalance/index.ts

```diff
@@ -1,12 +1,14 @@
-import { ethers } from "ethers";
+import { ethers, providers } from "ethers";
 import { getInteger } from "../getInteger";
-import { getJsonRpcProvider } from "../getJsonRpcProvider";
 import IERC20 from "../../abi/IERC20.json";
 import PairContract from "../../abi/PairContract.json";
 import { addresses } from "../../constants";
+import { getJsonRpcProvider } from "../getJsonRpcProvider";
 
-const getOwnedBCTFromSLP = async (adr: "bctUsdcLp" | "klimaBctLp") => {
-  const provider = getJsonRpcProvider();
+const getOwnedBCTFromSLP = async (
+  adr: "bctUsdcLp" | "klimaBctLp",
+  provider: providers.JsonRpcProvider
+) => {
```

Comment:
Please refactor to params syntax:
https://github.com/KlimaDAO/klimadao/wiki#prefer-params-objects-over-multiple-arguments